BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -362,6 +362,10 @@ extension ConvivaAnalytics: BitmovinPlayerListenerDelegate {
         client.adEnd(sessionKey)
     }
     #endif
+
+    func onDestroy() {
+        endSession()
```

Comment:
If a source was loaded, `endSession` was already called from the `onSourceUnloaded` listener. Maybe the `endSession` method should be implemented in a way that it does not execute again if the session is already ended.